charts/spv-wallet-stack/charts/block-headers-service/values.yaml

```diff
@@ -22,7 +22,7 @@ global:
 replicaCount: 1
 
 image:
-  repository: bsvb/block-headers-service
+  repository: public.ecr.aws/k9t8b9f1/block-headers-service
```

Comment:
IMHO it would be better to:
- make it easier to choose repository organisation
- setup cloud formation to set the repository organisation to the AWS ECR
- by default use dockerhub bsvb organisation, because 
  - we're publishing new images for each release to dockerhub bsvb
  - dockerhub doesn't need docker login for public repositories - so it's easier for people to use anywhere

We can set this up if this is priority.

Reply:
Great. Yes please do that, and it is of high priority.